### DISCUSSION TITLE
Email app coming?

### DISCUSSION THREAD

**Discussion by BD9Max** [FossifyOrg/General-Discussion#137] [General]
In the plans?

**Comment by naveensingh** [comment]
No?

**Comment by BD9Max** [comment]
Ok?

**Comment by BD9Max** [comment]
You could fork K-9 mail / thunderbird-android

**Reply by inson1** [reply]
@DB9-max And why dont you just use that?
Plus https://github.com/orgs/FossifyOrg/discussions/68
